Update catalog-products-add-edit.md (#180)
PrestaShop/prestashop-specs: back-office/catalog/products/catalog-products-add-edit.md

* **Product list**: Link that displays the list of all products filtered as in the catalog, with their id, name, price, quantity.       
If you click on the id or the name, you arrive on the product page in basic settings tab.       
If you click on the price, you arrive on the product page in pricing tab.       
If you click on the quantity, you arrive on the product page in the second tab which is: Quantities for a standard product without combinations or a pack, Combinations for a standard product with combination, Virtual product for a virtual product.       

> When the quantity is equal or less than 0 the quantity is displayed with red backgound.

Reviewer: ```suggestion
When the quantity is equal or less than 0 the quantity is displayed with a red background.
```

* **Save**: Button to save all changes made in all tabs.      
Keyboard shortcut: ALT + SHIFT + S.      
If it’s saved with success, there is a green success message “Settings updated”. Otherwise, there is an error message.

> * **Duplicate**: When you click on this button, it saves and duplicate the current product \(the duplicated product will be offline\). The duplicated product has all the product information (product associated, carriers configuration, associated suppliers, infromation of all the combinaition and the associated files) of the copied product.

Reviewer: ```suggestion
* **Duplicate**: When you click on this button, it saves and duplicates the current product \(the duplicated product will be offline\). The duplicated product has all the product information (related products, shipping configuration, associated suppliers, combinations, and attached files) of the copied product.
```

Reviewer: I've just updated the list with the same terminology used in the product page to make it clearer. ;-)

* **Category tree**: When the user creates a new product, the tree is open for the first 2 levels of categories \([issue 14955](https://github.com/PrestaShop/PrestaShop/issues/14955)\).       
When the user clicks on "collapse" it folds the whole tree. The user can also click on each parent to folds / unfolds its subcategories. When the user clicks on "expend" the whole tree is opened.       
Categories are sorted by level AND position:       
Categ 1 level 1 position 1       
Categ 3 level 1 position 2       
Categ 2 level 1 position 3       
Categ 2-2 level 2 position 1       
Categ 2-1 level 2 position 2       
On the left of the category's name is display a checkbox, if it is checked then the category is associated to the product. The last category can't be removed unchecked.
* **Main category**: Radio button to select the main category of the product. There can only be one.           
The main category is the one that will be in the product URL, before the product name and in the breadcrumb in front-office you have all parent categories up to the main one.

> If the category flaged as main category is unchecked in the tree or removed from the associated category, the associated category with the hightest position on the tree will be assgned automaticaly the main categoty.

Reviewer 2: ```suggestion
If the category flaged as main category is unchecked in the tree or removed from the associated category, the associated category with the hightest position on the tree will automaticaly be assgned  as the main category.
```

  Categ 3 level 1 position 2          
  Categ 2 level 1 position 3             
  Categ 2-2 level 2 position 1          
  Categ 2-1 level 2 position 2           

> If the list has elements with the same name, the lists displays the all the categaries with their ID. (ID - Name_category).

Reviewer 2: ```suggestion
If the list has elements with the same name, the lists displays all the categories with their ID. (ID - Name_category).
```

  * The cross to close the frame and unselect images
  * Caption field: You can enter a caption which will be applied to all images. If some selected images already had captions, they are replaced when clicking on Save.

    If more than one image is selected the checkbox "cover" disappears.
    

> The **Features section** is displayed if the Feature option in the Advanced Parameters / Performance / Optional features page enables otherwise the section is hidden.

Reviewer 2: ```suggestion
The **Features section** is displayed if the Feature option in Advanced Parameters > Performance > Optional features page is enabled. Otherwise the section is hidden.
```
